Header header

uint16[] gridInfo
uint16[] nodeGridInfo
